ddtrace/tracer/option_test.go

```diff
@@ -190,3 +191,17 @@ func TestServiceName(t *testing.T) {
 		assert.Equal("", globalconfig.ServiceName())
 	})
 }
+
+func TestGlobalTag(t *testing.T) {
+	t.Run("PresentInStatsdClient", func(t *testing.T) {
```

Comment:
```suggestion
	t.Run("statsd", func(t *testing.T) {
```
This should be enough, it will already be displayed as:
```
=== TestGlobalTag/statsd
```
... it's obvious what it is